assets/javascripts/discourse/routes/user-activity-my-reactions.js.es6

```diff
@@ -0,0 +1,26 @@
+import DiscourseRoute from "discourse/routes/discourse";
+import CustomReaction from "../models/discourse-reactions-custom-reaction";
+
+export default DiscourseRoute.extend({
+  userActionType: 17,
+  noContentHelpKey: "discourse_assigns.no_reactions",
+
+  model() {
+    return CustomReaction.findMyReactions(
+      this.modelFor("user").get("username_lower")
```

Comment:
```suggestion
      this.modelFor("user").username_lower
```

Reply:
I'm actually not sure here, what does modelFor returns if we don't get a model for the user ?